File: ftp_ls.py
import ftplib

def main():
    server = 'serwer194525.lh.pl'
    user = 'serwer194525'
    password = 'KochamAntygravity2026$'

    try:
        ftp = ftplib.FTP_TLS(server)
        ftp.login(user, password)
        ftp.prot_p()
        print("Connected!")
        print("Root dir:", ftp.nlst())
        try:
            print("public_html:", ftp.nlst("public_html"))
        except Exception as e:
            print("public_html error:", e)
        ftp.quit()
    except Exception as e:
        print(f"Error: {e}")

if __name__ == "__main__":
    main()

File: ftp_ls.js
const ftp = require("basic-ftp");

async function main() {
    const client = new ftp.Client();
    client.ftp.verbose = true;
    try {
        await client.access({
            host: "serwer194525.lh.pl",
            user: "serwer194525",
            password: "KochamAntygravity2026$",
            secure: true
        });
        console.log("Connected");
        console.log("Root directory:");
        console.log(await client.list());
        
        console.log("public_html directory:");
        console.log(await client.list("public_html"));
    }
    catch(err) {
        console.log(err);
    }
    client.close();
}

main();
